frontend/src/store/Settings/index.js

```diff
@@ -0,0 +1,57 @@
+import { observable, action, makeObservable } from 'mobx'
+
+class Settings {
+  constructor() {
+    makeObservable(this)
+  }
+
+  @observable widgets = [
+    { label: 'Screen', id: '1', selected: true },
+    { label: 'Battery', id: '2', selected: true },
+    { label: 'Robot\'s Speed', id: '3', selected: true },
+    { label: 'Additional actions', id: '4', selected: true },
+  ]
+
+  @observable robotTypes = [
```

Comment:
Just to clarify: is it a stub for the data that should come from BE?

Reply:
correct, it is a stub